enclave/src/rpc/io_handler_extensions.rs

```diff
@@ -0,0 +1,54 @@
+/*
+    Copyright 2019 Supercomputing Systems AG
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+*/
+
+pub extern crate alloc;
+use alloc::{borrow::ToOwned, str, string::String, vec::Vec};
+
+use jsonrpc_core::IoHandler;
+
+pub fn get_all_rpc_methods_string(io_handler: &IoHandler) -> String {
+    let method_string = io_handler
+        .iter()
+        .map(|rp_tuple| rp_tuple.0.to_owned())
+        .collect::<Vec<String>>()
+        .join(", ");
+
+    format!("methods: [{}]", method_string)
+}
```

Comment:
Pulls all registered method names out of the IoHandler and joins them into a single string